src/popup/components/InputField.vue

```diff
@@ -96,6 +99,7 @@
 
     <div
       v-if="showMessage"
+      :style="{ visibility: (hideError && focused) ? 'hidden' : undefined }"
```

Comment:
For the visibility change I suggest to use `v-show`. But to be honest I don't see a reason to use both `v-if` and visibility change. Maybe just use the `v-if`.